bridges/BakaUpdatesMangaReleasesBridge.php

```diff
@@ -3,82 +3,154 @@ class BakaUpdatesMangaReleasesBridge extends BridgeAbstract {
 	const NAME = 'Baka Updates Manga Releases';
 	const URI = 'https://www.mangaupdates.com/';
 	const DESCRIPTION = 'Get the latest series releases';
-	const MAINTAINER = 'fulmeek';
-	const PARAMETERS = array(array(
-		'series_id' => array(
-			'name'		=> 'Series ID',
-			'type'		=> 'number',
-			'required'	=> true,
-			'exampleValue'	=> '12345'
+	const MAINTAINER = 'fulmeek, KamaleiZestri';
+	const PARAMETERS = array(
+		'By series' => array(
+			'series_id' => array(
+				'name'		=> 'Series ID',
+				'type'		=> 'number',
+				'required'	=> true,
+				'exampleValue'	=> '12345'
+			)
+		),
+		'By list' => array(
+			'list_id' => array(
+				'name' 		=> 'List ID and Type',
+				'type' 		=> 'text',
+				'required' 	=> true,
+				'exampleValue'	=> '123456&list=read'
+			)
 		)
-	));
+	);
 	const LIMIT_COLS = 5;
 	const LIMIT_ITEMS = 10;
+	const RELEASES_URL = 'https://www.mangaupdates.com/releases.html';
 
 	private $feedName = '';
 
 	public function collectData() {
-		$html = getSimpleHTMLDOM($this->getURI())
-			or returnServerError('Series not found');
+		if($this -> queriedContext == 'By series') {
+			$html = getSimpleHTMLDOM($this->getURI())
+				or returnServerError('Series not found');
 
-		// content is an unstructured pile of divs, ugly to parse
-		$cols = $html->find('div#main_content div.row > div.text');
-		if (!$cols)
-			returnServerError('No releases');
+			// content is an unstructured pile of divs, ugly to parse
+			$cols = $html->find('div#main_content div.row > div.text');
+			if (!$cols)
+				returnServerError('No releases');
 
-		$rows = array_slice(
-			array_chunk($cols, self::LIMIT_COLS), 0, self::LIMIT_ITEMS
-		);
+			$rows = array_slice(
+				array_chunk($cols, self::LIMIT_COLS), 0, self::LIMIT_ITEMS
+			);
 
-		if (isset($rows[0][1])) {
```

Comment:
Too much changes with tabulation. Could you please do this instead:
1. rename collectData to something like collectDataBySeries
2. add method like collectDataByList, where you add your feature
3. create new method collectData with checks context and runs other methods depending on context